src/main/java/io/jenkins/plugins/PipelineEventGraphListener.java

```diff
@@ -99,14 +105,27 @@ private PipelineEvent asPipelineEvent(FlowNode flowNode) throws IOException, Int
                 .buildId(run.getId())
                 .timestamp(run.getTime())
                 .error(Optional.ofNullable(flowNode.getError()).map(ErrorAction::getError))
-                .branch(checkoutAction.map(CheckoutAction::getRepoUrl).orElse(null))
```

Comment:
don't do this..`"null"` isn't a real null value.  Instead, put `@JsonInclude(Include.NON_NULL)` on the `LiatrioV1Pipeline` class